disinfect_slam/disinfect_slam.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <string>
+#include <thread>
+#include <mutex>
+#include <atomic>
+#include <vector>
+#include <utility>
+
+#include <opencv2/core/core.hpp>
+
+#include <openvslam/system.h>
+#include <openvslam/publish/map_publisher.h>
+#include <popl.hpp>
+#include <spdlog/spdlog.h>
+#include <yaml-cpp/yaml.h>
+
+#include "cameras/l515.h"
+#include "cameras/zed_native.h"
+#include "modules/slam_module.h"
+#include "modules/tsdf_module.h"
+#include "segmentation/inference.h"
+#include "utils/time.hpp"
+#include "utils/gl/renderer_base.h"
+#include "utils/cuda/errors.cuh"
+#include "utils/rotation_math/pose_manager.h"
+#include "utils/config_reader.hpp"
+#include "modules/renderer_module.h"
```

Comment:
Do we need all those headers for this?

Reply:
Headers were changed in a commit yet to be merged: https://github.com/alvinsunyixiao/disinfect-slam/blob/roger/ros_interfacing/disinfect_slam/disinfect_slam.h